web/impact/impact/v1/helpers/model_helper.py

```diff
@@ -24,6 +24,10 @@ def merge_fields(field, extension):
     return result
 
 
+def json_schema(type):
+    return {"json-schema": {"type": type}}
```

Comment:
rename 'type' so the name doesnt clash with a built-in python method

Reply:
Fixed.